terms/abs/abs.md

```diff
@@ -0,0 +1,51 @@
+## Syntax
+```cs
+Math.Abs(value)
+```
+Math.Abs() is a static method of the Math class. It takes a single parameter of type int, long, float, double, or decimal. It returns the absolute value of the number.
+
+## Example:
+Find the absolute value of the number -5.
+```cs
+int x = -5;
+int y = Math.Abs(x);
+Console.WriteLine(y);
+```
+
+Codebyte Exmaple:
+```codebyte/csharp  
+using System;
+public class Program
+{
+    public static void Main()
+    {
+        int x = -5;
+        int y = Math.Abs(x);
+        Console.WriteLine(y);
+    }
+}
+```
+
+The output of the above code can be seen such:
```

Comment:
The rest of this can be omitted. Codebytes don't require an output block, and the code here is confusing as to what its trying to illustrate, especially as it is not runnable as written.